combat-trainer.lic

```diff
@@ -38,19 +37,19 @@ class SetupProcess
           break_summoned_weapon?(checkleft)
         end
       else
-        EquipmentManager.instance.stow_weapon(@weapon_training[game_state.weapon_skill])
+        EquipmentManager.instance.stow_weapon(game_state.weapon_training[game_state.weapon_skill])
```

Comment:
This is the next thing I have my eye on. There's some weirdness with how `game_state.weapon_name` differs from `game_state.weapon_training[game_state.weapon_skill]` that I need to investigate first.